src/main/webapp/WEB-INF/views/UserManagement.jsp

```diff
@@ -0,0 +1,142 @@
+                            </div>
+
+                            <div class="row">
+                                <div class="form-actions floatRight">
+                                    <input type="submit" value="{{!userController.user.id ? 'Add' : 'Update'}}"
+                                        class="btn btn-primary btn-sm" ng-disabled="myForm.$invalid">
+                                    <button type="button" ng-click="userController.reset()"
+                                        class="btn btn-warning btn-sm" ng-disabled="myForm.$pristine">Reset
+                                        Form</button>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="panel panel-default">
+                    <!-- Default panel contents -->
+                    <div class="panel-heading"><span class="lead">List of Users </span></div>
+                    <div class="tablecontainer">
+                        <table class="table table-hover">
+                            <thead>
+                                <tr>
+                                    <th>ID</th>
+                                    <th>Name</th>
+                                    <th>Address</th>
+                                    <th>Email</th>
+                                    <th width="20%"></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr ng-repeat="u in userController.users">
+                                    <td><span ng-bind="u.id"></span></td>
+                                    <td><span ng-bind="u.username"></span></td>
+                                    <td><span ng-bind="u.address"></span></td>
+                                    <td><span ng-bind="u.email"></span></td>
+                                    <td>
+                                        <button type="button" ng-click="userController.edit(u.id)"
+                                            class="btn btn-success custom-width">Edit</button> <button type="button"
+                                            ng-click="userController.remove(u.id)"
+                                            class="btn btn-danger custom-width">Remove</button>
+                                    </td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+
+            <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.4/angular.js"></script>
+            <script src="<c:url value='/static/js/app.js' />"></script>
+            <script src="<c:url value='/static/js/service/user_service.js' />"></script>
+            <script src="<c:url value='/static/js/controller/user_controller.js' />"></script>
```

Comment:
similar comments,  also consider a .tag file for head, and a .tag file for footer to make consistent partials in jsp